raiden/api/python.py

```diff
@@ -815,3 +817,35 @@ def get_blockchain_events_channel(
             ))
         returned_events.sort(key=lambda evt: evt.get('block_number'), reverse=True)
         return returned_events
+
+    def create_monitoring_request(
+            self,
+            payment_identifier: typing.PaymentID,
+            reward_amount: typing.TokenAmount,
+    ) -> RequestMonitoring:
+        # find `channel_state` with signed balance proof for `payment_identifier`
```

Comment:
nitpick: I don't think this comment is needed. The function is well named: `et_state_change_with_balance_proof_by_payment_id`.

What would be nice is to add a docstring to `create_monitoring_request()` explaining what the function accomplishes